core/src/Kokkos_CopyViews.hpp

```diff
@@ -3533,24 +3533,10 @@ create_mirror(const Kokkos::View<T, P...>& src,
       "not explicitly allow padding!");
 
   Kokkos::LayoutStride layout;
-
-  layout.dimension[0] = src.extent(0);
-  layout.dimension[1] = src.extent(1);
-  layout.dimension[2] = src.extent(2);
-  layout.dimension[3] = src.extent(3);
-  layout.dimension[4] = src.extent(4);
-  layout.dimension[5] = src.extent(5);
-  layout.dimension[6] = src.extent(6);
-  layout.dimension[7] = src.extent(7);
-
-  layout.stride[0] = src.stride_0();
-  layout.stride[1] = src.stride_1();
-  layout.stride[2] = src.stride_2();
-  layout.stride[3] = src.stride_3();
-  layout.stride[4] = src.stride_4();
-  layout.stride[5] = src.stride_5();
-  layout.stride[6] = src.stride_6();
-  layout.stride[7] = src.stride_7();
+  for (unsigned int i = 0; i < src_type::rank; ++i) {
+    layout.dimension[i] = src.extent(i);
+    layout.stride[i]    = src.stride(i);
+  }
```

Comment:
Would there be something wrong with just `Kokkos::LayoutStride layout = src.layout();`, or just passing `src.layout()` as an argument and forgoing the copy? We know from the `enable_if` that the source layout is `LayoutStride`

Reply:
No, I think we don't even need to specialize for `LayoutStride`.